__tests__/index.test.js

```diff
@@ -24,3 +24,54 @@ describe("/api/topics tests", () => {
     return request(app).get("/api/unknownendpoint").expect(404);
   });
 });
+
+describe("/api tests", () => {
+  test("documents available endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        expect(response.body).toHaveProperty("GET /api");
+        expect(response.body).toHaveProperty("GET /api/topics");
+      });
+  });
+});
+describe("/api/articles/:article_id tests", () => {
+  test("200: sends a single article to the client", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article.article_id).toBe(1);
+        expect(response.body.article).toHaveProperty("author");
+        expect(response.body.article).toHaveProperty("title");
+        expect(response.body.article).toHaveProperty("body");
+        expect(response.body.article).toHaveProperty("topic");
+        expect(response.body.article).toHaveProperty("created_at");
+        expect(response.body.article).toHaveProperty("votes");
+        expect(response.body.article).toHaveProperty("article_img_url");
+      });
+  });
+  ("");
+  test("400: responds with bad request for invalid article id", () => {
+    return request(app)
+      .get("/api/articles/one")
+      .expect(400)
+      .then((response) => {
+        expect(response.body.message).toBe("Bad request.");
+      });
+  });
+  test("404: responds with not found for valid but non-existent article id", () => {
+    return request(app)
+      .get("/api/articles/11111111111111111111111")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.message).toBe(
+          "The data with the given id was not found."
+        );
+      });
+  });
+  test("404: missing article id", () => {
+    return request(app).get("/api/articles/").expect(404);
+  });
+});
```

Comment:
This test is a standard 404 for no route found so should be picked up/ tested by your 404 not found test.